railties/lib/rails/generators/test_unit/model/templates/fixtures.yml.tt

```diff
@@ -4,7 +4,8 @@
 <%= name %>:
 <% attributes.each do |attribute| -%>
   <%- if attribute.password_digest? -%>
-  password_digest: <%%= BCrypt::Password.create("secret") %>
+  # Generated with BCrypt::Password.create("secret")
+  password_digest: <%= BCrypt::Password.create("secret") %>
```

Comment:
You might need some escaping to not conflict with YAML here (not sure what's the range of characters used by bcrypt). A good way to do that is `.to_json`.